start_esp/start_esp.py

```diff
@@ -872,6 +885,21 @@ def make_argparser():
 
     parser.add_argument('--compute_platform_override', default=None, help=argparse.SUPPRESS)
 
+    parser.add_argument('--enable_grpc_backend_ssl',
+        action='store_true', help='''Enable SSL for gRPC backend.''')
+
+    parser.add_argument('--grpc_backend_ssl_root_certs_file',
+        default='/etc/nginx/trusted-ca-certificates.crt',
+        help='''Set the file path for gRPC backend SSL root certificates.''')
```

Comment:
nit: delete "Set the"? 
'''File path for gRPC backend SSL root certificates'''

the same for the other two grpc flags
also feel free to leave it as it

Reply:
Done. Thanks